source/command-monitoring/tests/find.json

```diff
@@ -270,6 +270,135 @@
     },
     {
       "description": "A successful find event with a getmore and killcursors",
+      "ignore_if_server_version_greater_than": 3.0,
```

Comment:
Is this different enough from "A successful find event with a getmore" to be worth having?